src/tools.rs

```diff
@@ -396,6 +400,30 @@ pub async fn cache_dir() -> Result<PathBuf> {
     Ok(path)
 }
 
+async fn get_http_client(root_certificate: &Option<PathBuf>, accept_invalid_certs: bool) -> Result<reqwest::Client> {
```

Comment:
I think it would be great if we could combine those arguments into some `HttpClientOptions` struct.

Reply:
Yes, that's up next. I was just getting the merge conflicts out of the way.